packages/mc-html-template/src/process-headers.spec.js

```diff
@@ -0,0 +1,98 @@
+import processConfig from './process-headers';
+
+const defaultApplicationConfig = {
+  name: 'Application test',
+  entryPointUriPath: 'test',
+  cloudIdentifier: 'test',
+  env: {
+    env: 'test',
+  },
+  headers: {},
+};
+
+describe('header string', () => {
+  describe('when value is a string', () => {
+    it('should convert to a header string', () => {
+      const testApplicationConfig = {
+        ...defaultApplicationConfig,
+        headers: {
+          featurePolicies: {
+            microphone: "'none'",
+            camera: ["'https://example.com'"],
+          },
+        },
+      };
+
+      const processedApplicationConfig = processConfig(testApplicationConfig);
+
+      expect(
+        processedApplicationConfig['Feature-Policy']
+      ).toMatchInlineSnapshot(
```

Comment:
I find this more convenient than `toEqual` cause it would mean copy pasting while this nicely shows the diff to accept under test.